src/modules/party/dto/create-party-request.dto.ts

```diff
@@ -1,62 +1,76 @@
-import { ApiProperty } from '@nestjs/swagger';
-import { EXAMPLES, PROPERTIES } from '@ukef/constants';
-import { IsIn, IsISO8601, IsOptional, IsString, Length, Matches } from 'class-validator';
+import { ENUMS, EXAMPLES, PROPERTIES } from '@ukef/constants';
+import { ValidatedDateOnlyApiProperty } from '@ukef/decorators/validated-date-only-api-property.decorator';
+import { ValidatedStringApiProperty } from '@ukef/decorators/validated-string-api-property.decorator';
 
-export type CreatePartyRequest = CreatePartyRequestItem[];
+export type CreatePartyRequestDto = CreatePartyRequestItem[];
 
 export class CreatePartyRequestItem {
-  @ApiProperty({ description: 'The UKEF ID for the party. Should contain just digits.', minLength: 8, maxLength: 8, example: '00291013', pattern: '/^d{8}$/' })
-  @Length(8, 8, {
-    message: 'alternateIdentifier must be exactly 8 characters',
-  })
-  @Matches(/^\d+$/, {
-    message: 'alternateIdentifier must only contain digits',
+  @ValidatedStringApiProperty({
+    description: 'The UKEF ID for the party.',
+    length: 8,
+    example: EXAMPLES.PARTY_ID,
```

Comment:
This example is an ACBS party id, not a UKEF party alternate identifier

Reply:
I've added `EXAMPLES.PARTY_ALTERNATE_ID`, with value `'12345678'`